courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2501 @@
+{
```

Comment:
<p>Do you use numpy anywhere? I didn't see it.</p><p> </p><p>print statement should go after imports.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
You are correct, I'm not using numpy -- old habits!
Moved print statement to last line in cell, after imports.